### PR TITLE
refactor(theme): restore the external link icon color from previous version

### PR DESCRIPTION
**Pull request by zonemeen**
I think the color of the original external link icon is more appropriate.

image

vs 

image

### CODE REVIEW

**Comment by brc-dd** [comment]
What about the links in text?

Current:

image

After this PR:

image

It was looking a bit weird to me, that's why I changed the color. We probably can introduce a color variable and override it only for VPLink (so that c-text-3 is used on sidebar/navbar links). WDYT?

**Comment by zonemeen** [comment]
I think this implementation is feasible. I've submitted new code, take a look.

File: src/client/theme-default/components/VPDoc.vue [hunk 1]
```diff
@@ -205,5 +205,6 @@ const pageName = computed(() =>
 
 .external-link-icon-enabled :is(.vp-doc a[href*='://'], .vp-doc a[target='_blank'])::after {
   content: '';
+  color: currentColor;
```
Comment (brc-dd) [inline src/client/theme-default/components/VPDoc.vue:208]:
```suggestion
```

File: src/client/theme-default/styles/components/vp-doc.css [hunk 2]
```diff
@@ -534,6 +534,7 @@
   width: 11px;
   height: 11px;
   background: currentColor;
+  color: var(--vp-c-text-3);
```
Comment (brc-dd) [inline src/client/theme-default/styles/components/vp-doc.css:537]:
```suggestion
  background: var(--vp-c-text-3);
```

**Review by brc-dd — APPROVED** [review]

**Pull request merged**